lib/server-code/runners/cloud-master.js

```diff
@@ -19,7 +19,7 @@ module.exports = async function runMaster(opts) {
   const lowPriorityThreshold = opts.workers.lowPriorityThreshold
   const taskRequests = {}
 
-  const messageBroker = new MessagesBroker(opts.backendless.msgBroker, lowPriorityThreshold ? 2 : 1)
+  const messageBroker = new MessagesBroker(opts, lowPriorityThreshold ? 2 : 1, true)
```

Comment:
I assume we can pass all the necessary options right here:

```
const messageBroker = new MessagesBroker({
  connection: opts.backendless.msgBroker,
  compression: opts.compression.prod,
  gettersCount: lowPriorityThreshold ? 2 : 1,
```
what do you think?

Reply:
Agree. Done